src/llvm/ext/llvm_ext.cc

```diff
@@ -351,9 +351,19 @@ void LLVMMetadataReplaceAllUsesWith2(
 void LLVMExtSetCurrentDebugLocation(
   LLVMBuilderRef Bref, unsigned Line, unsigned Col, LLVMMetadataRef Scope,
   LLVMMetadataRef InlinedAt) {
+#if LLVM_VERSION_GE(12, 0)
+  if (!Scope)
+    unwrap(Bref)->SetCurrentDebugLocation(DebugLoc());
```

Comment:
Is this okay? It seems. Line and col are ignored?

Reply:
It's looks like it probably doesn't matter so much in practice because if we don't have a scope, Crystal is clearing out the debug location anyway: https://github.com/crystal-lang/crystal/blob/dd40a2442fa186add8a82b74edb14a90aa1dae05/src/compiler/crystal/codegen/debug.cr#L441

Reply:
Sounds good, thanks!